index.html

```diff
@@ -1443,6 +1482,66 @@ <h2>Handling members of our section going online or offline</h2>
         LoopEnd --> LoopStart
     </div>
 
+	<h2>Handling join requests from new nodes</h2>
+	<div class=description>
+		This is the flow for section members to handle nodes that attempt to join the network.
+		When an incoming connection is detected, the section member will notify the rest of the
+		section and request a relocation for the new node. Then, everything will progress the same
+		way it does with a regular relocation.
+
+        <button class="collapsible">Parsec::NewCandidate</button>
+        <div class="content">
+			<p>An event containing the name of the new candidate.<br />
+			A new candidate will only contact a single member of the section. Thus, this will
+			only require a single vote to accumulate. Once such a single event gets consensused,
+			all nodes that receive it via poll() will act upon it.
+            </p>
+        </div>
+        <button class="collapsible">State::InitialRelocation</button>
+        <div class="content">
+			<p>A special node state, meaning that the node is only awaiting its initial relocation.
+            </p>
+        </div>
+        <button class="collapsible">can_accept</button>
+        <div class="content">
+			<p>A function that returns "true" if the number of nodes in State::InitialRelocation
+            we are connected to is less than the maximum allowed (to be defined).</p>
+        </div>
+	</div>
+	<div class="mermaid">
+		graph TB
+		StartRespondToJoinRequests --> LoopStart
+		style StartRespondToJoinRequests fill:#f9f,stroke:#333,stroke-width:4px
+
+		WaitFor(("Wait for 10:"))
+		LoopStart --> WaitFor
+
+		WaitFor --"RPC"--> RPC
+		RPC(("RPC"))
+
+		WaitFor --"Parsec consensus"--> Consensus
+		Consensus(("Consensus"))
+
+		RPC --"Rpc::RequestRelocate"--> Check
+		Check(("Check"))
+		
+		Check --"can_accept"--> VoteNewNode
```

Comment:
We can have that check here as well, to avoid Parsec Spam, but we need to check on consensus, otherwise we will accept more nodes that we want.